src/settings/ConditionsSettings.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { Settings } from '@folio/stripes/smart-components';
+import { stripesConnect } from '@folio/stripes/core';
+
+import Conditions from './patronBlocks/Conditions/Conditions';
+
+class ConditionsSettings extends Component {
+  static manifest = Object.freeze({
+    patronBlockConditions: {
+      type: 'okapi',
+      records: 'patronBlockConditions',
+      path: 'patron-block-conditions',
+    },
+  });
+
+  static propTypes = {
+    resources: PropTypes.shape({
+      patronBlockConditions: PropTypes.shape({
+        records: PropTypes.arrayOf(PropTypes.object),
+      }),
+    }),
+    mutator: PropTypes.shape({
+      patronBlockConditions: PropTypes.shape({
+        GET: PropTypes.func,
```

Comment:
Mark it as isRequired. (mutator, patronBlockConditions, GET)

Reply:
Fixed